File: problems/leetcode/150-interview/151-Reverse-Words-in-a-String.java
class Solution {
    public String reverseWords(String s) {
        String[] str = s.trim().split(\\\\\s+\);
        String res = \\;
        for (int i = str.length - 1; i > 0; i--) {
            res += str[i] + \ \;
        }
        return res + str[0];
    }
}

File: problems/leetcode/LeetCode-75/151-Reverse-Words-in-a-String.cs
public class Solution {
    public string ReverseWords(string s) {
        string reverse = \\;
        var words = s.Trim().Split(' ');
        foreach (var word in words){
            if (word.Length!=0)
                reverse = word + \ \ + reverse;
        }
        return reverse.Trim();  
    }
}